Project.toml

```diff
@@ -12,7 +12,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
-JuMP = "^0.21.0"
+JuMP = "~0.22"
 MathOptInterface = "^0.9.1, 0.10"
```

Comment:
v0.9 should be removed here. For instance, `MOI.SolveTimeSec` is define in v0.10 but not v0.9.